runtimes/dune

```diff
@@ -15,10 +15,17 @@
    (python/** with_prefix runtime_python)))
  (section lib))
 
-; Rescript runtime
+;; ; Rescript runtime -- disabled
```

Comment:
Why disabled ? Should we reenable it?

Reply:
I don't think so, we could remove the whole directory; the only difference is on the visibility of the thing I guess.
In any case, probably better left for a specific PR.